### PR TITLE
PMO list and KH2 enemy list update

### PR DESCRIPTION
**Pull request by osdanova**
Added a Dictionary section to BBS's docs. Worlds file moved to dictionaries and PMO dictionary added.
Updated the enemy list for KH2's battle editor.

### CODE REVIEW

**Review by Truthkey — CHANGES_REQUESTED** [review]
In `models.md` make sure to separate the models by category. Add subsections with ### to separate B/G/F and the like. Also, if possible have links with [id] (#Category-you-want) without the space in the Category section.

**Comment by osdanova** [comment]
Didn't divide it at first so we had a continuous list that can be copied and reordered, but it's easy enough to just remove the sections if needed when doing so. Done.

File: docs/bbs/index.md [hunk 1]
```diff
@@ -16,3 +15,7 @@
 * Other files
     * [font.arc, fonten.arc](font.md)
 
+## Dictionaries
+
+* [Worlds](dictionary/worlds.md)
```
Comment (Delta-473) [inline docs/bbs/index.md:20]: Is moving worlds.md to the dictionary dir coming in an other commit?

Comment (osdanova) [reply docs/bbs/index.md:20]: I actually forgot... Thank you.

**Comment by Truthkey** [comment]
You are still missing a lot of models in this list. For example, none of the **P** from **BD** world are included.
For example: P01BD00, P51BD00...

Also, I recommend you to include the 2.5 exclusive P93EX00 which is Armored Terra with a cape.

**Comment by osdanova** [comment]
> You are still missing a lot of models in this list. For example, none of the **P** from **BD** world are included.
> For example: P01BD00, P51BD00...
> 
> Also, I recommend you to include the 2.5 exclusive P93EX00 which is Armored Terra with a cape and the JP exclusive W14VE00 which is Light Seeker.

I've added Terra, thank you! As for the rest:
About light seeker, are you sure that's the id? It's the same filename as Frolic Flame. (2FM)
About P01BD00, P51BD00... those are arc files and not pmo files, thus I have not listed them.

Could you explain these 2 points to me in the Discord Server? Thank you.

**Comment by osdanova** [comment]
Light Seeker added. Change "Models" to "PMO files" because arc files also contain models and are not listed.

**Review by Truthkey — APPROVED** [review]

**Pull request merged**